### ISSUE TITLE
Product Variant Bulk Action Price

### ISSUE THREAD

**Issue by YodasMyDad** [Merchello/Merchello#2027]
Be really nice to be able to bulk update variant prices by a %. As most of the time you get a % increase from a supplier. And this way each price could be different but still only increase by X %

**Comment by mothman1** [comment]
Good point, but I would also vote for "import and export variants csv file" to at least update variants prices. For example, I have a product that links to four options. Each option contains three values. The result of such product variants is 81 variants - each one's price is different than the other.

**Comment by KShips** [comment]
the import from csv above would be great, prices and stock levels are something that can change frequently (especially stock when selling in store as well).
